.github/workflows/docsite-publish.yml

```diff
@@ -6,8 +6,13 @@ on:
     branches:
       - master
   workflow_dispatch:
-    branches:
-      - master
+    inputs:
+      sha:
+        description: The SHA-1 hash referring to the commit to check.
+        required: true
+      ref:
+        description: The head branch associated with the pull request.
```

Comment:
why is the `ref` necessary ? shouldn't the commit sha be enough ?

Reply:
Apparently the [chromaui action requires both these inputs](https://github.com/chromaui/action#triggering-from-workflow_dispatch).

Reply:
can we add this intel as a comment in the workflow ?

Reply:
Comments added to describe the requirements.